server/controllers/studentController.js

```diff
@@ -32,8 +34,19 @@ const createStudent = async (req, res) => {
 
     // add doc to db
     const { fName, age, location, sStatus, email, password } = req.body;
+    //handeling errors 
+    //still empty
+
+
     try {
-        const student = await Student.create({ fName, age, location,sStatus, email, password })
+        const student_id = req.student._id
```

Comment:
same here, probably not needed within try/catch and underscore before the id. usually an underscore gets used for stuff you want to skip (like a fn parameter) or params you want to throw away after. If you learned it otherwise I would be interested in the source :)

Reply:
it because by default, Mongoose adds an _id property to the schemas
so i put it like this to match with Mongoose